### PR TITLE
add clearml & wandb

### PR DESCRIPTION
**Pull request by Louis-Dupont**
Relies on https://github.com/Deci-AI/super-gradients/pull/685 for the registry

### CODE REVIEW

File: documentation/source/experiment_monitoring.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+### Tensorboard
+**requirements**: None
+
+Tensorboard is natively integrated into the training and validation steps. You can find how to use it in [this section](TODO:add_link_to_our_tn_description).
+
+
+### Weights & Biases
+**requirements**:
+- Install `wandb`
```
Comment (ofrimasad) [inline documentation/source/experiment_monitoring.md:17]: before this line:
register to W&B and make sure you have a user name and token to connect to the service

File: documentation/source/experiment_monitoring.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+trainer = Trainer("experiment_name")
+model = ...
+
+training_params = {
+    ...                             # Your training params
+    "sg_logger": "wandb_sg_logger", # Weights&Biases Logger, see class super_gradients.common.sg_loggers.wandb_sg_logger.WandBSGLogger for details
+    "sg_logger_params":             # Params that will be passes to __init__ of the logger super_gradients.common.sg_loggers.wandb_sg_logger.WandBSGLogger
+      {
+        "project_name": "project_name", # W&B project name
```
Comment (ofrimasad) [inline documentation/source/experiment_monitoring.md:32]:
```
entity:
api_server:   # in case your experiment tracking is not hosted at wandb servers
```

Comment (Louis-Dupont) [reply documentation/source/experiment_monitoring.md:30]: Added in 5168bb65f56074a9aa1cac07d0a28b19a55c7107

File: documentation/source/experiment_monitoring.md (hunk 1 above)
Comment (Louis-Dupont) [reply documentation/source/experiment_monitoring.md:14]: I added something like that 5168bb65f56074a9aa1cac07d0a28b19a55c7107

**Comment by Louis-Dupont** [comment]
> * please add a section about building and using your own SGLogger.
> * add in the opening section a note that says that you can integrate any monitoring tool into SG with minimal code changes.
> * add links to the installation guides of wandb and clearml
> * add a section showing how to configure these parameters from YAML

Concerning the wandb/cleaml links, I already had it before, maybe it didnt show in your interface.
I added the other things you talked about in ccc83efec1ab8b2698af5e4b17735a31f138cd48

**Review by ofrimasad — COMMENTED** [review]
looks great. just a minor comment that needs to be edited/removed

File: documentation/source/experiment_monitoring.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+from PIL import Image
+from typing import Union
+
+from super_gradients.common.sg_loggers.base_sg_logger import BaseSGLogger
+from super_gradients.common.environment.ddp_utils import multi_process_safe
+
+
+@register_logger()
+class CustomSGLogger(BaseSGLogger):
+    """Logger responsible to push logs and tensorboard artifacts to Deci platform."""
```
[the remark this answers is not in the document]
Comment (Louis-Dupont) [reply documentation/source/experiment_monitoring.md]: Changed

**Review by ofrimasad — APPROVED** [review]
LGTM
